packages/Python/lldbsuite/test/lang/swift/resilience/mod.a.swift

```diff
@@ -12,14 +12,20 @@
 
 public struct S {
   public var a = 1
-  public var s1 = "i"
-  public var s2 = "am"
-  public var s3 = "large"
-  public var s4 = "!"
+  fileprivate var s1 = "i"
+  fileprivate var s2 = "am"
+  fileprivate var s3 = "large"
+  fileprivate var s4 = "!"
 
   public init() {}
 }
 
+var g_s = S()
```

Comment:
I would add tests to cover a few more cases:

- Tuple containing two structs (total size < 24 bytes)
- Tuple containing two structs (total size > 24 bytes)
- Fixed layout struct containing resilient struct (total size < 24 bytes)
- Fixed layout struct containing resilient struct (total size > 24 bytes)
- A struct that is < 24 bytes but not bitwise takable and hence is still out of line (eg, it contains a `weak var`)